### PR TITLE
Sample updates to get them to work with new versions of Gradle and build as part of our main build

### PR DESCRIPTION
**Pull request by ledina**
Sample updates to get them to work with new versions of Gradle and build as part of our main build

### CODE REVIEW

File: samples/gradle-multipart-sample/settings.gradle [hunk 1]
```diff
@@ -1,3 +1,14 @@
+pluginManagement {
+    repositories {
+        //mavenLocal() // enable this to test a local version published using publishToMavenLocal goal
+        mavenCentral()
+        maven {
+            name = "SonatypeSnapshots"
```
Comment (davenice) [inline samples/gradle-multipart-sample/settings.gradle:6]: is it definitely intended to enable snapshots, for the samples?

Comment (ledina) [reply samples/gradle-multipart-sample/settings.gradle:6]: We do it in Maven I believe, and it means we can easily test a snapshot when developing. Could comment it out if we think that is a problem.

Comment (davenice) [reply samples/gradle-multipart-sample/settings.gradle:6]: ok - I don't remember having it in there permanently. is there anything in doing this that would imply to users that it's the right thing to do?

Comment (ledina) [reply samples/gradle-multipart-sample/settings.gradle:6]: It enables users to use snapshots if they so wish. If there is a new feature they want to try and we haven't officially released it yet they can. I don't  mind too much, but thought best be persistent in both Maven/Gradle

Comment (ledina) [reply samples/gradle-multipart-sample/settings.gradle:6]: OK, one Maven sample has it in, the other doesn't. I have taken the approach of leaving it in but it is commented out.

**Review by davenice — APPROVED** [review]

**Pull request merged**